docker/dev/clone_db.sh

```
@@ -0,0 +1,42 @@
+
+# Restore the database from the dump file
+echo "Restoring the local database from the remote dump..."
+docker exec journeyplanner-db-1 bash -c 'mariadb -u$MYSQL_USER -p$MYSQL_PASSWORD $MYSQL_DATABASE < /var/lib/mysql/journeyplanner.sql'
+
+# Remove the dump file to clean up
+echo "Removing the temporary dump file from the container..."
+docker exec journeyplanner-db-1 bash -c 'rm /var/lib/mysql/journeyplanner.sql'
+
+# Run database migrations to ensure schema is up to date
+echo "Running database migrations to apply any pending changes..."
+cd ../../backend
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)